### PR TITLE
Add array messages and main function

### PR DESCRIPTION
**Pull request by fleurcodes**
These changes add the array messages and the main function that returns the final random fortune to the user

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1 +1,27 @@
...
+  "You will be successful in fulfilling your plans.", "Administration matters will be in favor.", "Proposals will get support.", "Take care of your loved ones.", "Business will be better.", "Health will be good.", "Morale will increase.", 
+  "Try to get the necessary work done.", "You'll earn financial achievements.", "Wealth will increase.", "Feel free to keep moving forward.", "Seize opportunities!", "Focus on work.", "Avoid procastination!", "Efforts will bear fruit.", 
+  "Desired results will be achieved.", "There will be a boom in career.", "Business will prosper.", "Be health-conscious.", "Be careful in judicial matters.", "Increase control over the budget.", "Be patient in personal matters.", 
+  "Avoid rumors.", "Do not fall into greed and temptation.", "Relationships will be strengthened.", "You should get regular health checkups.", "Use your wisdom and intelligence to solve problems.", "Work with dignity."];
+
+
+// Main function: Returns final random fortune to user
+const randomMessage = arr => {
+  const randomMessageIdx = Math.floor(Math.random() * arr.length);
+  const randomLuckyNumber = Math.floor(Math.random() * 15);
```
Comment (vickalchev) [inline app.js:17]: Do we also need to produce a lucky number or is it extra?
It looks like the lucky number is less than 15. Is this in the assignment?
No problem adding it, just curious.

File: app.js [hunk 2]
```diff
@@ -1 +1,27 @@
...
+  if (randomMessageIdx === 0) {
```
Comment (vickalchev) [inline app.js:18]: It looks like if the randomly selected index in the array is 0, then we also show a lucky number? Am I understanding this part correctly?

**Review by vickalchev — APPROVED** [review]

**Comment by vickalchev** [comment]
Hey Criss, I just tested your code and it works beautifully. Great job. I'm going to push it to the master.

File: app.js (hunk 2 above)
Comment (fleurcodes) [reply app.js:18]: Yes! Exactly :)

File: app.js (hunk 1 above)
Comment (fleurcodes) [reply app.js:17]: No, the lucky number is exclusively if you get the messages[0] which is "Your lucky number: ". I saw that a lot of horoscope web pages told you your lucky numbers of the day so i figured why not include as well a random lucky number in this project too! It doesn't comes with the assignment, i added it for fun :)

**Pull request merged**